### DISCUSSION TITLE
Can't stop bentoml serve

### DISCUSSION THREAD

**Discussion by hungcao0402** [bentoml/BentoML#3951] [Q&A] [answered]
I run bentoml serve and can't stop this service on port 3000

**Comment by frostming** [comment]
What does it outputs? is there any error message? And how do you stop it?

**Reply by hungcao0402** [reply]
I have run "bentoml serve"  and now it still run on port 3000. I can't kill process run on this port by "kill -9 pid".
Any way to stop this process?

**Reply by aarnphm** [reply]
Can you do `pgrep python3 | xargs kill -9`?

**Reply by frostming** [reply, marked as answer]
It is run under a process manager that will automatically pull up workers when they are dead. So you need to kill the process group or kill the manager process.

**Reply by hungcao0402** [reply]
I have fixed it, thanks for the help.
